src/openforms/formio/dynamic_config/tests/test_referentielijsten_config.py

```diff
@@ -424,6 +453,36 @@ def test_items_not_found(self):
             _("No results found from Referentielijsten API."),
         )
 
+    def test_items_expired(self):
+        component: SelectBoxesComponent = {
+            "key": "selectboxes",
+            "type": "selectboxes",
+            "label": "Selectboxes",
+            "values": [],
+            "dataType": "string",
+            "openForms": {
+                "code": "not_geldig_option",
+                "dataSrc": DataSrcOptions.referentielijsten,
+                "service": self.service.slug,
+                "translations": {},
+            },
+        }
+
+        with self.assertRaises(ServiceUnavailable):
+            register.update_config(component, submission=self.submission, data={})
```

Comment:
Is this correct behaviour? This was intended to be returned if the reference lists service is broken, but if it's actually up and running and just returns expired items, then I would expect the form to just be available as usual and not return a 503